kubernetes/cray-ims/templates/cray-ims-v2-image-customize-job-template.yaml

```diff
@@ -244,6 +242,19 @@ data:
           - name: admin-client-auth
             secret:
               secretName: "{{ .Values.keycloak.keycloak_admin_client_auth_secret_name }}"
+          - name: image-vol
+            image-vol:
+              name: image-vol
+              persistentVolumeClaim: 
+                claimName: image-vol-data-claim
+          persistentVolumeClaims:
+            data-claim:
+              name: data-claim
+              accessModes:
+              - ReadWriteOnce
+              resources:
+                requests:
+                  storage: 100Gi
```

Comment:
This should be based on template param similar to '$limit_gb' - that was set when this was a mem based volume to indicate the max size it could expand to. That gives the customer control in case it gets wildly huge. Maybe create a new template param 'pvc_size' that is like 5X expected job size?

https://github.com/Cray-HPE/ims/blob/develop/src/server/v2/resources/jobs.py#L687
https://github.com/Cray-HPE/ims/blob/develop/src/server/v3/resources/jobs.py#L693